### PR TITLE
Provide Console Use

### PR DESCRIPTION
**Pull request by 95CivicSi**
Included in this update
 - Proper console usage updates for several commands.
   - spawnpet
   - leveluppet
   - healpet
   - changepetname
   - togglepet
 - Modifications to sendConsoleError
   - Now takes 2 arguments.
   - 2nd argument is a boolean flag. True when the console is not allowed to use a command.  Defaults to false.
 - Fix errors when no arguments passed in on commands.
   - Commands should now send usage to the person who called the command with no arguments.
 - Updated language files with new error message for console updates

Users will need to update their default language files to avoid errors related to no-console-use messages.
  
`[Server thread/CRITICAL]: ErrorException: "Undefined index: no-console-use" (EXCEPTION) in "BlockPets/src/BlockHorizons/BlockPets/configurable/LanguageConfig" at line 41`  

If the user has not customized their messages, the easiest way to do this is to delete `./plugins/BlockPets/languages` and allow the plugin to repopulate at restart.

### CODE REVIEW

File: src/BlockHorizons/BlockPets/commands/PetCommand.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 						$sender->sendMessage(
 							TextFormat::GREEN . "/spawnpet <petType> [petName|select] [size] [isBaby] [player]: " . TextFormat::YELLOW . "Spawns a new pet with the given data." . PHP_EOL .
 							TextFormat::GREEN . "/removepet <name> [player]: " . TextFormat::YELLOW . "Removes the first pet with the given name, and checks for a player if given." . PHP_EOL .
-							TextFormat::GREEN . "/togglepet [all/name]: " . TextFormat::YELLOW . "Toggles your pets on/off, depending on the current state." . PHP_EOL .
+							TextFormat::GREEN . "/togglepet <pet name> [player]: " . TextFormat::YELLOW . "Toggles a pet on/off, depending on the current state." . PHP_EOL .
```
Comment (Sandertv) [inline src/BlockHorizons/BlockPets/commands/PetCommand.php]: Do note that people can use /togglepet all to toggle all pets, you might just want to keep that there.

File: src/BlockHorizons/BlockPets/commands/ChangePetNameCommand.php [hunk 2]
```diff
@@ -22,8 +22,14 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 			return true;
 		}
 
-		if(!$sender instanceof Player) {
+		if(!isset($args[0])){
```
Comment (Sandertv) [inline src/BlockHorizons/BlockPets/commands/ChangePetNameCommand.php]: There are quite a bunch of those tiny formatting problems. I'm fine with merging it and reformatting it later but it'd be very much appreciated if you could fix those. 😜

File: src/BlockHorizons/BlockPets/commands/PetCommand.php (hunk 1 above)
Comment (95CivicSi) [reply src/BlockHorizons/BlockPets/commands/PetCommand.php]: Duly Noted and corrected.

**Comment by Haxley** [comment]
Nice 💯

**Comment by 95CivicSi** [comment]
Thanks @Haxley!  Hopefully you'll find this useful too.

**Pull request merged**